Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.css

```diff
@@ -0,0 +1,24 @@
+footer {
```

Comment:
good job with styling, I like how you have your css segmented out as well for components like this